src/views/dashboard/components/header/header.tsx

```diff
@@ -1,107 +1,244 @@
-import React, { useEffect, useState } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-import { useUserStore } from '../../../../stores/user-store/user-store';
-import { aboutUsRoute, homeRoute, loginRoute } from '../../../../shared/routes/routes';
-import { useTranslation } from 'react-i18next';
-import { MenuTransaltion } from '../../context/menuTranslation';
-import { destinationRequest } from '../../../../services/api/destination/destinationService';
-import userIcon  from '../../../../assets/images/user.svg';
+import React, { useEffect, useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { useUserStore } from "../../../../stores/user-store/user-store";
+import {
+  aboutUsRoute,
+  homeRoute,
+  loginRoute,
+} from "../../../../shared/routes/routes";
+import { useTranslation } from "react-i18next";
+import { MenuTransaltion } from "../../context/menuTranslation";
+import { destinationRequest } from "../../../../services/api/destination/destinationService";
+import userIcon from "../../../../assets/images/user.svg";
 // @TODO change logo and zakynthos
-import logo from '../../../../assets/images/logo.svg';
-import zakynthos  from '../../../../assets/images/zakynthos-JPEG.jpg';
+import logo from "../../../../assets/images/logo.svg";
+import hamburgerMenu from "../../../../assets/images/menu-hamburger.svg";
+import close from "../../../../assets/images/remove.svg";
+import circle from "../../../../assets/images/circle.svg";
+import zakynthos from "../../../../assets/images/zakynthos-JPEG.jpg";
+import { Collapse } from "antd";
 
 import "./header.scss";
 
 const Header = () => {
-
   const history = useHistory();
   const { t: translate } = useTranslation();
   const [menuContentVisibility, setMenuContentVisibility] = useState(false);
+  const [menuMobileVisibility, setMenuMobileVisibility] = useState(false);
+  const [menuMobileStyle, setMenuMobileStyle] = useState({ maxHeight: "0rem" });
 
   const [destinations, setDestinations] = useState<any[]>([]);
 
-  const { state: {user} } = useUserStore();
+  const {
+    state: { user },
+  } = useUserStore();
   const { logoutUser } = useUserStore();
 
+  const { Panel } = Collapse;
+
   useEffect(() => {
     fetchDestinations();
-  },[]);
+  }, []);
 
   const fetchDestinations = async () => {
     const { data } = await destinationRequest();
     setDestinations(data);
-  }
+  };
 
   const handleMenuContent = (show: boolean) => {
     setMenuContentVisibility(show);
-  }
+  };
+
+  const handleMenuMobile = (value: boolean) => {
+    setMenuMobileVisibility(value);
+    if (value === true) {
+      setMenuMobileStyle({ maxHeight: "200rem" });
```

Comment:
also it would be easier to have setMenuMobileStyle({maxHeight: value ? '200rem' : '0rem'})